assets/js/atomic/utils/create-blocks-from-template.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Creates blocks for a given inner blocks Template.
+ *
+ * @param {Array} template Inner Blocks Template.
+ */
+export const createBlocksFromTemplate = ( template ) => {
+	return template.map( ( [ name, atts = {}, innerBlocks = [] ] ) => {
+		const children = innerBlocks
+			? createBlocksFromTemplate( innerBlocks )
+			: [];
+		return createBlock( name, atts, children );
```

Comment:
Cool! Were and @nerrad discussing this in slack the other day? 

I'm curious about whether this is standard/expected use of gutenberg APIs or if we have special needs. Might be worth adding a bit more info about this and our expectations for it. Is this something we would prefer to import from Gutenberg? How stable is `createBlock` API?

Reply:
Could also add JSDoc typedefs for the expected param shape and the return value.

Reply:
I think createBlock is stable, this code isn't actually new and is in the current production build - I've just moved it so reuse is easier. The difference is that this works recursively rather than over a flat list of blocks.